config/templates/pages/example.xml

```diff
@@ -116,19 +116,25 @@
 
             <params>
                 <param name="values" type="collection">
+                    <param name="">
```

Comment:
I think this option should never needed to be set by the dev itself. It should always be there if there is no `default_value` set.

Reply:
its also okay for me to remove the option in the `example.xml` and just adjust the javascript in this PR 🙂

Reply:
is okay for me to keep the `example.xml` at current state.

Reply:
> It should always be there if there is no default_value set.

At the moment if no `default_value` param is used, no option for unselecting is possible. So I guess this comment is obsolete?